File: WPServer/WPProjectServer/WPProjectServer/Logic/ClientLogic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WPProjectServer.DatabaseObjects;
using WPProjectServer.Logic.SubsystemLogic;

namespace WPProjectServer.Logic
{
    public class ClientLogic : IClientLogic
    {
        public CopyStatus RentCopy(string copyCode, string userCode)
        {
            return LogicFactory.GetLogic<BookIOSubLogic>().RentCopy(copyCode, userCode);
        }

        public CopyStatus ReturnCopy(string copyCode)
        {
            return LogicFactory.GetLogic<BookIOSubLogic>().Return(copyCode);
        }
    }
}

File: WPWeb/Library/src/main/java/controllers/ClientLogic.java
package controllers;

import models.CopyStatus;
import models.User;

/**
 * Created by Szymon on 2015-10-13.
 */


public class ClientLogic implements IClientLogic
{
        LogicFactory logicFactory;

        public ClientLogic()
        {
                logicFactory = new LogicFactory();
        }

        public CopyStatus RentCopy(String copyCode, String userCode) throws Exception
        {
                return logicFactory.GetBookIOLogic().RentCopy(copyCode, userCode);
        }

        public CopyStatus ReturnCopy(String copyCode) throws Exception
        {
                return logicFactory.GetBookIOLogic().Return(copyCode);
        }

        public boolean RegisterUser(User user)
        {
                return logicFactory.GetUsersLogic().RegisterUser(user);
        }


}